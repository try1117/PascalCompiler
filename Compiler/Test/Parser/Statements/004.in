program test;
var
  a: array[1..10] of integer;
  idx: integer = 0;
begin
  a[idx] := 25;
end.
