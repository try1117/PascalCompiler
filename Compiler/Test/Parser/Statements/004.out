Lexical exception in position (6,8) - Index out of range
